packages/material-ui/src/Button/Button.js

```diff
@@ -319,7 +338,10 @@ Button.propTypes = {
   /**
    * If `true`, the ripple effect will be disabled.
    */
-  disableRipple: PropTypes.bool,
+  disableRipple: deprecated(
+    PropTypes.bool,
+    'Focus ripple will be removed because it does not follow Material design.',
```

Comment:
I don't understand. We do want to keep the ripple. It's the pulsating focusing ripple we want to remove no?

Reply:
You're right. That should be at `disableFocusRipple`.